Core/src/main/java/com/intellectualcrafters/plot/commands/Alias.java

```diff
@@ -2,27 +2,83 @@
 
 import com.intellectualcrafters.plot.PS;
 import com.intellectualcrafters.plot.config.C;
+import com.intellectualcrafters.plot.object.Location;
 import com.intellectualcrafters.plot.object.Plot;
 import com.intellectualcrafters.plot.object.PlotPlayer;
 import com.intellectualcrafters.plot.object.StringWrapper;
 import com.intellectualcrafters.plot.util.MainUtil;
+import com.intellectualcrafters.plot.util.Permissions;
+import com.intellectualcrafters.plot.util.StringMan;
 import com.intellectualcrafters.plot.util.UUIDHandler;
 import com.plotsquared.general.commands.CommandDeclaration;
 
+import de.schlichtherle.util.Arrays;
```

Comment:
What is this? 😕

Reply:
@Sauilitired
what would be better for you addind the permisson again and use the
plots.set.alias as obsolete permission and plots.set.alias as new permisson.
or
leave the original permisson plots.set.alias and remove the new one plots.alias.set
Note: should we change the plots.alias.remove to plots.remove.alias or should we leave it as it was

Reply:
I will remove this that was a misstake